doc/source/testing.rst

```diff
@@ -45,19 +45,33 @@ prior to merging or submitting a pull request.
 
     $ tox
 
+List available scenarios
+------------------------
+
+List all available scenarios. This is useful to target specific Python and
+Ansible version for the functional and unit tests.
+
+    $ tox -l
+
+
 Unit
 ----
 
 Run all unit tests with coverage.
 
 .. code-block:: bash
 
-    $ tox -t unit
+    $ tox -e $(tox -l | grep unit | paste -d, -s -)
```

Comment:
Just
```suggestion
    $ tox -e unit
```
that's what [factors](https://tox.readthedocs.io/en/latest/config.html?highlight=factor#generating-environments-conditional-settings) are for.

Reply:
Sorry but it would execute unit only on the current python version, see https://github.com/ansible/molecule/issues/1764#issuecomment-465992799